src/RVRP_data.jl

```diff
@@ -80,31 +80,39 @@ mutable struct Request # can be
     delivery_time_windows::Vector{FlexibleRange}
 end
 
+mutable struct VehicleCharacteristics
+    of_vehicle::Dict{String,Float64} # defined only if measured at the vehicle level; for string id key associated with properties or capacity measures that need to be checked on the vehicle, as for instance weight, value, volume
+    of_compartments::Dict{String,Dict{String,Float64}}  # defined only if measured at the compartment level; for string id key associated with properties or capacity measures that need to be checked on the vehicle, as for instance weight, value, volume, ... For each such property, the Dictionary specifies the capacity for each compartment id key.
+end
+
 mutable struct VehicleCategory
     id::String
-    vehicle_capacities::Dict{String,Float64} # defined only if measured at the vehicle level; for string id key associated with properties capacity measures that need to be checked on the vehicle, as for instance weight, value, volume
-    compartment_capacities::Dict{String,Dict{String,Float64}} # defined only if measured at the compartment level; for string id key associated with properties capacity measures that need to be checked on the vehicle, as for instance weight, value, volume, ... For each such property, the Dictionary specifies the capacity for each compartment id key.
-    vehicle_properties::Dict{String,Float64} # defined only if measured at the vehicle level; for string id key associated with properties that need to be checked on the vehicle (such as the same check applies to all the compartments), as for instance to ability to cary liquids or  refrigerated product.
-    compartment_properties::Dict{String,Dict{String,Float64}} #  defined only if measured at the compartment level; for string id key associated with properties that need to be check on the comparments such as  max weight, max length, refrigerated product, .... For each such property, the Dictionary specifies the capacity for each compartment id key.
+    capacity_measures::VehicleCharacteristics
+    vehicle_properties::VehicleCharacteristics
     loading_option::Int # 0 = no restriction (=default), 1 = one request per compartment, 2 = removable compartment separation (note that product conflicts are measured within a compartment)
     energy_interval_lengths::Vector{Float64} # at index i, the length of the i-th energy interval. empty if no recharging.
 end
 
+mutable struct WorkPeriod
```

Comment:
@FD-V I was talking to @IssamT yesterday and we had a doubt if it is ok or not to have a globar WorkPeriod for HomogeneousVehicleSet

Reply:
@IssamT